lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -353,6 +355,9 @@ class BarChartRodData with EquatableMixin {
   /// If you want to have a rounded rod, set this value.
   final BorderRadius? borderRadius;
 
+  /// If you want to have dashed border, set this value.
+  final List<int>? dashArray;
```

Comment:
I think it makes more sense if we use `borderDashArray` here. What do you think?

Reply:
Ah true, dashArray would be a bit confusing on its own.

Reply:
Renamed to borderDashArray, resolving conversation